components/cam/src/physics/cam/clubb_intr.F90

```diff
@@ -2616,7 +2711,12 @@ subroutine clubb_surface (state, ptend, ztodt, cam_in, ustar, obklen)
     
     do i = 1, ncol
        th(i) = state%t(i,pver)*state%exner(i,pver)         ! diagnose potential temperature
-       thv(i) = th(i)*(1._r8+zvir*state%q(i,pver,ixq))  ! diagnose virtual potential temperature
+       if (eamv2) then
+         thv(i) = th(i)*(1._r8+zvir*state%q(i,pver,ixq) & ! PMA corrects virtual potential temperature formula
+                       - state%q(i,pver,ixcldliq))
```

Comment:
need to get ixcldliq before using in this subroutine clubb_surface. thv fix in subroutine clubb_tend_cam is fine.